demo/repl-module.js

```diff
@@ -181,7 +181,7 @@ function onFailure(error, metadata) {
     output.setValue(metadata.transcoded);
   hasError = true;
   errorElement.hidden = false;
-  errorElement.textContent = error.stack || error;
+  errorElement.textContent = error.stack || error.join('\n');
```

Comment:
You are right that this is not sufficient.

There are 2 cases. The first one is that a runtime error happens. Then, error will be what was thrown. The other case is an error coming from traceur and in that case it is an array.

Unfortunately there is no way to distinguish throwing an array from parse errors handled by traceur.

This whole area needs some cleanup.